invenio/legacy/bibclassify/__init__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Legacy BibClassify."""
+
+import warnings
+
+from invenio.utils.deprecation import RemovedInInvenio22Warning
+
+warnings.warn("Legacy BibClassify will be removed in 2.2. Please check "
+              "'invenio.modules.classifier' module.",
+              RemovedInInvenio22Warning)
```

Comment:
Yes, and that's true for most other legacy modules.  The deprecation warning there basically says: if someone's looking in `invenio.legacy.foo`, they had better look into `invenio.modules.bar` instead.  It is true that the later one may stop working once the former one is removed... so `bar` has to be either seriously amended or even taken away and released independently, about which this PR does not say anything.